editor/src/clj/editor/pipeline/bob.clj

```diff
@@ -118,56 +114,11 @@
                                (progress/make-cancellable-indeterminate msg)
                                progress/done))))))))
 
-(defn- ->graph-resource-scanner [ws]
-  (let [res-map (->> (g/node-value ws :resource-map)
-                  (map (fn [[key val]] [(subs key 1) val]))
-                  (into {}))]
-    (reify IResourceScanner
-      (openInputStream ^InputStream [this path]
-        (when-let [r (get res-map path)]
-          (io/input-stream r)))
-      (exists [this path]
-        (if-let [r (get res-map path)]
-          (resource/exists? r)
-          false))
-      (isFile [this path]
-        (if-let [r (get res-map path)]
-          (= (resource/source-type r) :file)
-          false))
-      (scan [this pattern]
-        (let [res (->> res-map
-                    (map first)
-                    (filter #(PathUtil/wildcardMatch % pattern))
-                    set)]
-          res)))))
-
```

Comment:
There is no need for this code since it's used only for loading built-ins, and Bob already has an instrument to do it.